lib/iron_bank/csv.rb

```diff
@@ -4,23 +4,30 @@ module IronBank
   # A custom CSV converter
   #
   class CSV < ::CSV
+    DECIMAL_INTEGER_REGEX = /^[+-]?\d+$/.freeze
+    DECIMAL_FLOAT_REGEX   = /^[+-]?(?:\d*\.|\.\d*)\d*$/.freeze
+
     CSV::Converters[:decimal_integer] = lambda { |field|
```

Comment:
I do know why `reek` or `rubocop` are not picking it up, but can we use the `do` syntax for a multiline block?

```suggestion
    CSV::Converters[:decimal_integer] = lambda do |field|
```